ynput/ayon-applications
# Icons: Update flame icon

## Changelog Description
update flame icon
![image](https://github.com/user-attachments/assets/a8af4170-3193-4e9b-9147-e9ea1012da72)


## Testing notes:

1. create  and upload the addon
2. check it in launcher
